workers/migrations/0015-create_default_indexes.js

```diff
@@ -78,6 +78,15 @@ var indexes = {
       "_id": 1
     },
     "ns": "koding.jStackTemplates"
+  }, {
```

Comment:
not running it on prod does not allow us to update migration files, also we wont be running these first migrations in prod, since they are not meant for prod. 

migration files should be written for keeping development env runnning flawlessly, so i shouldnt be need to run buildservices to have migrations applied. so it wont work.